processor/tile_grpc.go

```diff
@@ -58,22 +59,41 @@ func (gi *GeoRasterGRPC) Run() {
 		return
 	}
 
+	g0 := grans[0]
+	effectivePoolSize := int(math.Ceil(float64(len(grans)) / float64(g0.GrpcConcLimit)))
```

Comment:
There is no need to convert `len(grans)` to `float64`. Leave it as an integral type and Go will do the right thing if either the numerator or the denominator in a division are a floating point type.

Reply:
Initially I did exactly what you proposed but got compiler error "invalid operation: len(grans) / float64(g0.ConfigPayLoad.GrpcConcLimit) (mismatched types int and float64)". I think golang is pretty strict about its operands. So we just need to cast both ints into float64 before taking a float point division.

Reply:
Fair enough. I fell into the trap myself with a test program:

```
package main
const x = 10
func main() {
  print(x / float64(1))
}
```

Of course, in Go, the compiler decides on the type of `x`, so chose `float64`!